core-library/src/main/java/org/silverpeas/core/util/MultiSilverpeasBundle.java

```diff
@@ -32,6 +32,9 @@
 import org.silverpeas.core.util.StringUtil;
 
 import java.text.ParseException;
+import java.time.Instant;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.time.Instant'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)